### PR TITLE
Add key file for SFTP and backup rotation note for mount point

### PR DESCRIPTION
**Pull request by tom-andrews**
Added key file location for SFTP server configuration.
Added (lack of) backup rotation note for Local mount point.

### CODE REVIEW

File: layouts/partials/embeds/backup-locations.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Before you specify to backup to an SFTP server, make sure that:
         to the SFTP server.
     - **Use a custom key** - Generate a TLS key pair for the SFTP server, copy the private key to
         the **SSH Private Key** box, and copy the public key to the SFTP server.
+    - The key shoud be added to ~/.ssh/authorized_keys of the user used for backup on SFTP server.
```
Comment (bmansheim) [inline layouts/partials/embeds/backup-locations.md]: @ushachar Should we tell customers where to put the key?

File: layouts/partials/embeds/backup-locations.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ Before you specify to backup to a local mount point, make sure that:
 - The node has network connectivity to the destination server of the mount point.
 - The `redislabs:redislabs` user has read and write priviledges on the local mount point
 and on the destination server.
+- Periodic backup doesn't include rotation functionality. It's your responsibility to enforce and maintain backup retention policy.
```
Comment (bmansheim) [inline layouts/partials/embeds/backup-locations.md]: @RoeyPrat Each backup file has a unique name, right?

File: layouts/partials/embeds/backup-locations.md (hunk 1 above)
Comment (ushachar) [reply layouts/partials/embeds/backup-locations.md]: No - it depends on which SFTP server they are running, and not something we can control / know.

File: layouts/partials/embeds/backup-locations.md (hunk 2 above)
Comment (yaeltzirulnikov) [reply layouts/partials/embeds/backup-locations.md]: yes, each backup file has a timestamp.

**Pull request merged**